src/app/games/diaballik/tests/diaballik.component.spec.ts

```diff
@@ -5,8 +5,8 @@ import { DiaballikMove, DiaballikPass, DiaballikTranslation } from '../Diaballik
 import { MGPOptional } from 'src/app/utils/MGPOptional';
 import { Coord } from 'src/app/jscaip/Coord';
 import { DiaballikPiece, DiaballikState } from '../DiaballikState';
-import { DiaballikFailure } from '../DiaballikRules';
 import { RulesFailure } from 'src/app/jscaip/RulesFailure';
+import { DiaballikFailure } from '../DiaballikFailure';
```

Comment:
noté au mauvais endroit mais, le tutoriel du step 4 est en tant que jaune, mais je suis quand même en haut au lieu d'être en bas!

Reply:
tiens c'est pas normal ça, ça le fait sur d'autres jeux ?

Reply:
Y en a pas moulte ou on joue Light, mais je crois pas non, fallait faire le retournage simplement, non ?

Reply:
mais le faire où ? Si faut faire un retournage à la main quand on est P1, c'est un bug du tutorial-game-wrapper, on est d'accord ?

Reply:
Kamisado est un switcheur aussi, et il a du Player.ONE qui est en bas quand c'est son tour !

Par contre autre question: quand on joue humain contre humain, Player.ZERO reste en bas tout le temps hein ?

Reply:
MENSONGE! C'est pas un bug en fait, ou il s'est réglé tout seul !